knowledge-base/actions/step-security/wait-for-secrets/action-security.yml

```diff
@@ -0,0 +1,3 @@
+name: "Get MFA Secrets"
+#GITHUB_TOKEN Not Used
```

Comment:
Please add `id-token: write` permission `to authenticate to the StepSecurity API`, view [here](https://github.com/step-security/wait-for-secrets#aws-secrets)